Remotion/Web/Development.WebTesting/ScreenshotCreation/BrowserContentLocators/ChromeBrowserContentLocator.cs

```diff
@@ -27,9 +27,9 @@
 namespace Remotion.Web.Development.WebTesting.ScreenshotCreation.BrowserContentLocators
```

Comment:
I'm commenting this here because I can't comment on the changed reference screenshot directly. After changing ScreenshotTest_TableContainer.png the screenshot test now fails in Firefox (at least locally on my machine)

Chrome, Edge and even IE seemed fine

Reply:
The failing test must be caused by your setup, as Firefox uses `Firefox.any.ScreenshotTest_TableContainer.png` as reference image.